install.sh

```diff
@@ -1,43 +1,52 @@
+#!/bin/bash
+
+# exit on fail
+set -e
+
 #chromebrew directories
 OWNER="skycocker"
 REPO="chromebrew"
 BRANCH="master"
-URL="https://raw.githubusercontent.com/$OWNER/$REPO/$BRANCH"
-CREW_PREFIX=${CREW_PREFIX:-/usr/local}
-CREW_LIB_PATH=$CREW_PREFIX/lib/crew/
-CREW_CONFIG_PATH=$CREW_PREFIX/etc/crew/
-CREW_BREW_DIR=$CREW_PREFIX/tmp/crew/
-CREW_DEST_DIR=$CREW_BREW_DIR/dest
-CREW_PACKAGES_PATH=$CREW_LIB_PATH/packages
-
-architecture=$(uname -m)
-
-if [ $EUID -eq 0 ]; then
-  echo 'Chromebrew should not be run as root.'
+URL="https://raw.githubusercontent.com/${OWNER}/${REPO}/${BRANCH}"
+CREW_PREFIX="${CREW_PREFIX:-/usr/local}"
+CREW_LIB_PATH="${CREW_PREFIX}/lib/crew/"
+CREW_CONFIG_PATH="${CREW_PREFIX}/etc/crew/"
+CREW_BREW_DIR="${CREW_PREFIX}/tmp/crew/"
+CREW_DEST_DIR="${CREW_BREW_DIR}/dest"
+CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
+
+ARCH="$(uname -m)"
+
+if [ "${EUID}" == "0" ]; then
+  echo 'Chromebrew should not be installed or run as root.'
   exit 1;
 fi
 
-case "$architecture" in
+case "${ARCH}" in
 "i686"|"x86_64"|"armv7l"|"aarch64")
+  LIB_SUFFIX=
+  [ "${ARCH}" == "x86_64" ] && LIB_SUFFIX='64'
   ;;
 *)
   echo 'Your device is not supported by Chromebrew yet.'
   exit 1;;
 esac
 
 # This will allow things to work without sudo
-sudo chown -R `id -u`:`id -g` "${CREW_PREFIX}"
+sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
 
 # prepare directories
-for dir in $CREW_LIB_PATH $CREW_CONFIG_PATH $CREW_CONFIG_PATH/meta $CREW_BREW_DIR $CREW_DEST_DIR $CREW_PACKAGES_PATH; do
-  mkdir -p $dir
+for dir in "${CREW_CONFIG_PATH}/meta" "${CREW_DEST_DIR}" "${CREW_PACKAGES_PATH}"; do
```

Comment:
Are you sure the `CREW_LIB_PATH` and `CREW_CONFIG_PATH` directories are not necessary?

Reply:
Yes. Those directories are created recursively with the `-p` option to `mkdir`.